### PR TITLE
logodesign

### PR DESCRIPTION
**Pull request by baranpirincal**
Good luck with your new logo design!

Best Regards
Baran

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +0,0 @@
...
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
```
Comment (AndreiMisiukevich) [inline LICENSE]: @baranpirincal could you please preserve this file as is ? cuz it's project license :) thanks

**Comment by AndreiMisiukevich** [comment]
@baranpirincal could you please resolve this small issue ? :)

**Comment by baranpirincal** [comment]
@AndreiMisiukevich is it okay now? :)

**Comment by AndreiMisiukevich** [comment]
@baranpirincal you removed this file https://github.com/AndreiMisiukevich/HotReload/blob/master/LICENSE

Preserve it please

Could you please revert this?

**Comment by baranpirincal** [comment]
> @baranpirincal you removed this file https://github.com/AndreiMisiukevich/HotReload/blob/master/LICENSE
> 
> Preserve it please
> 
> Could you please revert this?

is it okay now? sorry I am not good on github :/

**Comment by AndreiMisiukevich** [comment]
@baranpirincal look :)

This file
image

should has name "LICENSE" all capitalized

And should be here: 
image

At the root folder

Does it make sense?

**Comment by baranpirincal** [comment]
> @baranpirincal look :)
> 
> This file
> image
> 
> should has name "LICENSE" all capitalized
> 
> And should be here:
> image
> 
> At the root folder
> 
> Does it make sense?

I did, could you please check it?

**Pull request merged**